src/System.Management.Automation/engine/lang/parserutils.cs

```diff
@@ -1521,6 +1527,81 @@ public bool MoveNext()
             return true;
         }
     }
+
+    /// <summary>
+    /// The simple enumerator class is used for the range operator '..'
+    /// in expressions like 'A'..'B' | ForEach-Object { $_ }
+    /// </summary>
+    internal class CharRangeEnumerator : IEnumerator
+    {
+        private int _lowerBound;
+
+        internal int LowerBound
+        {
+            get { return _lowerBound; }
+        }
+
+        private int _upperBound;
+
+        internal int UpperBound
+        {
+            get { return _upperBound; }
+        }
+
+        private int _current;
+
+        Object IEnumerator.Current
+        {
+            get { return Current; }
+        }
+
+        public string Current
+        {
+            get { return new String((char)_current, 1); }
+        }
+
+        internal int CurrentValue
+        {
+            get { return _current; }
+        }
+
+        private int _increment = 1;
```

Comment:
Fields not backing public properties should be defined before properties . https://github.com/DotNetAnalyzers/StyleCopAnalyzers/blob/master/documentation/SA1202.md

Reply:
@markekraus I reformatted the class.